spec/models/user_status_spec.rb

```diff
@@ -7,7 +7,7 @@
 RSpec.describe UserStatus, type: :model do
   USER_STATUSES = %i[urgent_to_call accommodated in_office in_trip at_home no_response declined].freeze
   describe 'attributes' do
-    subject { described_class.new(status: STATUSES[0]) }
+    subject { described_class.new(status: USER_STATUSES[0]) }
```

Comment:
This change was added in the scope of this PR, but if your PR will be merged earlier, I fix it
https://github.com/vova/war_shelter/pull/27